frontend/src/components/trail-list/trails-list.test.tsx

```diff
@@ -55,4 +55,11 @@ describe('TrailList', () => {
       screen.getByText(`${trail.surveys_quantity} Survey`)
     ).toBeInTheDocument();
   });
+
+  it('should open a specific trail', async () => {
+    render(<TrailList data={trails} />);
+    const trailItem = screen.getByText(trails[0].name);
+    fireEvent.click(trailItem);
+    expect(trailItem).toBeInTheDocument();
+  });
```

Comment:
I think this test isn't doing what it's supposed to do. 

I suggest you to put move the current expect  before the `fireEvent` call and add another expect after the click to check if something new occurred with the execution of the event.

Reply:
How we can check if a new page open?

Reply:
you can mock the react-router method and check if it's called with the expected URL

Reply:
Fixed, thanks @andersonmalheiro !